### ISSUE TITLE
VTIMEZONE format in Microsoft Outlook

### ISSUE THREAD

**Issue by andi-wr** [iCalcreator/iCalcreator#86]
Hello,

it seems that Microsoft Outlook ignores the VTIMEZONE section generated by your ical-creator:

```
BEGIN:VTIMEZONE
TZID:Europe/Berlin
X-LIC-LOCATION:Europe/Berlin
BEGIN:STANDARD
TZNAME:CET
DTSTART:20191027T030000
TZOFFSETFROM:+0200
TZOFFSETTO:+0100
RDATE:20201025T030000
END:STANDARD
BEGIN:DAYLIGHT
TZNAME:CEST
DTSTART:20190331T020000
TZOFFSETFROM:+0100
TZOFFSETTO:+0200
RDATE:20200329T020000
RDATE:20210328T020000
END:DAYLIGHT
END:VTIMEZONE
```

I have a recurring event (weekly), starting In STANDARD (CET) time and ending in DAYLIGHT (CEST) time. After importing the ICS into Microsoft Outlook, all events within DAYLIGHT time are shifted by one hour. When I open the recurring event, it says " (UTC+01:00) West-Central Africa" instead of  "(UTC+01:00) Amsterdam, Berlin, Bern, Rom, Stockholm, Wien" (which is my timezone).

When I replace the VTIMEZONE section with another format starting in 1970 with a RRULE (last sunday in october / march), generated e.g. by Google Calendar, it works fine:

```
BEGIN:VTIMEZONE
TZID:Europe/Berlin
BEGIN:DAYLIGHT
TZOFFSETFROM:+0100
TZOFFSETTO:+0200
TZNAME:CEST
DTSTART:19700329T020000
RRULE:FREQ=YEARLY;BYDAY=-1SU;BYMONTH=3
END:DAYLIGHT
BEGIN:STANDARD
TZOFFSETFROM:+0200
TZOFFSETTO:+0100
TZNAME:CET
DTSTART:19701025T030000
RRULE:FREQ=YEARLY;BYDAY=-1SU;BYMONTH=10
END:STANDARD
END:VTIMEZONE
```

Other Ical-Creators I tested generate the second format, too. What's the reason for you to choose the first format?

By the way, e.g. Thunderbird has no problems with your format. ;-)

Thank you very much in advance!

**Comment by iCalcreator** [comment]
Well, both are valid ical.

iCalcreator is building up a dynamic VTIMEZONE, 
starting before required date or first calendar event DTSTART,
followed by the next STANDARD/DAYLIGHTs.

The other solution would imply building up at static
list of timezones and corresponding RRULEs, starting
from 1970 and onward.

I'll look into the issue.

**Comment by bobvandevijver** [comment]
This is still an issue for the desktop version, but when using the browser version of Outlook the iCal is imported just fine (and also displaced correctly in the desktop version).

**Comment by iCalcreator** [comment]
Fixed in the  2.41.61 release.
